src/node_platform.h

```diff
@@ -74,7 +76,11 @@ class PerIsolatePlatformData {
   uv_async_t* flush_tasks_ = nullptr;
   TaskQueue<v8::Task> foreground_tasks_;
   TaskQueue<DelayedTask> foreground_delayed_tasks_;
-  std::vector<DelayedTask*> scheduled_delayed_tasks_;
+
+  // Use a custom deleter because libuv needs to close the handle first.
+  typedef std::unique_ptr<DelayedTask, std::function<void(DelayedTask*)>>
+      DelayedScheduledTask;
```

Comment:
Can you call this `DelayedTaskPointer` or something like that? Just to make clear it’s still a pointer and it’s not a conceptually different kind of thing